core/controllers/editor.py

```diff
@@ -39,6 +39,7 @@
 from core.domain import user_services
 from core.domain import value_generators_domain
 from core.platform import models
+from constants import CONSTANTS
```

Comment:
OK. I'm fine with making an exception here, and also with not doing it at module-level.